src/backend/marsha/core/api/portability_request.py

```diff
@@ -101,13 +101,15 @@ def get_queryset(self):
             return PortabilityRequest.objects.regarding_user_id(
                 self.request.user.id,
                 include_owned_requests=True,
-            )
+            ).annotate_allow_action(self.request.user.id)
```

Comment:
In your body for commit `6a7b47b7980296cd04ec929c628498243b5e868e`

```suggest
s/Provide a new filed/Provide a new field`
```

And

`requests to profice information to the frontend`. Provide ?